pkgs/tools/misc/hwatch/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub, fetchpatch, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hwatch";
+  version = "0.3.5";
+
+  src = fetchFromGitHub {
+    owner = "blacknon";
+    repo = pname;
+    rev = "d86479f3b101be7c7b4ed0da3f008b5f2bf583fe";
```

Comment:
Why are we not using the version revision?

Reply:
Adjusted to "rev = version"